doc.json

```diff
@@ -414,68 +414,6 @@
         }
       }
     },
-    "/publication-{id}/file-upload": {
```

Comment:
nevermind the doc files. I ran `npm run doc` to make sure the doc would generate correctly. 
I really should have those file in the .gitignore because they will be generated again when I merge this into master.